client/my-sites/patterns/components/header/index.tsx

```diff
@@ -1,23 +1,180 @@
+			OPTIONS.home.description
+		),
+	},
+	header: {
+		title: translate( 'WordPress Header Patterns', OPTIONS.category.title ),
+		header: translate( 'Get a head start with patterns', OPTIONS.category.header ),
+		description: translate(
+			"Use predefined and beautiful patterns to decorate the header of your site, and catch your visitors' attention.",
+			OPTIONS.category.description
+		),
+	},
+	footer: {
+		title: translate( 'WordPress Footer Patterns', OPTIONS.category.title ),
+		header: translate( 'Build a strong foundation with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Choose from our collection of patterns to add essential information and navigation to the bottom of your site.',
+			OPTIONS.category.description
+		),
+	},
+	about: {
+		title: translate( 'WordPress About Patterns', OPTIONS.category.title ),
+		header: translate( 'Tell your story with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Add one of these patterns to your site to convey your mission and values compellingly.',
+			OPTIONS.category.description
+		),
+	},
+	posts: {
+		title: translate( 'WordPress Blog Post Patterns', OPTIONS.category.title ),
+		header: translate( 'Get more readers with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Take advantage of this selection of patterns that make your writing pop and keep readers engaged.',
+			OPTIONS.category.description
+		),
+	},
+	contact: {
+		title: translate( 'WordPress Contact Patterns', OPTIONS.category.title ),
+		header: translate( 'Connect with your audience using patterns', OPTIONS.category.header ),
+		description: translate(
+			'Pick from this set of patterns to add a contact form and a map to your site.',
+			OPTIONS.category.description
+		),
+	},
+	events: {
+		title: translate( 'WordPress Events Patterns', OPTIONS.category.title ),
+		header: translate( 'Promote events with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Highlight upcoming events on your site with patterns that grab attention and inform.',
+			OPTIONS.category.description
+		),
+	},
+	gallery: {
+		title: translate( 'WordPress Gallery Patterns', OPTIONS.category.title ),
+		header: translate( 'Stunning galleries with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Pick from our set of patterns to showcase your visuals with elegance.',
+			OPTIONS.category.description
+		),
+	},
+	intro: {
+		title: translate( 'WordPress Intro Patterns', OPTIONS.category.title ),
+		header: translate( 'Captivating introductions with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Make a good first impression with patterns that welcome visitors and guide them through your site.',
+			OPTIONS.category.description
+		),
+	},
+	menu: {
+		title: translate( 'WordPress Menu Patterns', OPTIONS.category.title ),
+		header: translate( 'Craft delightful menus with patterns', OPTIONS.category.header ),
```

Comment:
```suggestion
		title: translate( 'WordPress Restaurant Menu Patterns', OPTIONS.category.title ),
		header: translate( 'Craft delightful restaurant menus with patterns', OPTIONS.category.header ),
```
I feel we should mention "restaurant" here, too, and not just in the description. "WordPress menu" will probably make people think of navigation menus first and foremost.